qa-tests-backend/src/test/groovy/AdmissionControllerNoImageScanTest.groovy

```diff
@@ -124,11 +123,11 @@ class AdmissionControllerNoImageScanTest extends BaseSpecification {
                     orchestrator.deleteDeployment(deployment)
                     deleted = true
                 } catch (NullPointerException ignore) {
-                    println "Caught NPE while deleting deployment, retrying in 1s..."
+                    log.info "Caught NPE while deleting deployment, retrying in 1s..."
                 }
             }
             if (!deleted) {
-                println "Warning: failed to delete deployment. Subsequent tests may be affected ..."
+                log.warn "failed to delete deployment. Subsequent tests may be affected ..."
```

Comment:
Prior to this change, I believe most, if not all log messages were sentence case. We should make sure that there is consistency in the new way, sentence case or no sentence case. Now, the above info is a sentence case and this warning is not.